openmdao/components/tests/test_balance_comp.py

```diff
@@ -59,6 +59,54 @@ def test_scalar_example(self):
 
         assert_check_partials(cpd, atol=1e-5, rtol=1e-5)
 
+    def test_balance_comp_with_units_kwarg(self):
```

Comment:
not a big deal, but you could have tested both things in a single test ...

Reply:
Changed